packages/node_modules/@webex/internal-plugin-device/test/unit/spec/features/feature-model.js

```diff
@@ -56,6 +56,117 @@ describe('plugin-devices', () => {
       });
     });
 
+    describe('#parse()', () => {
+      let fixture;
+      let rto;
+
+      beforeEach('initialize the feature model', () => {
+        featureModel = new FeatureModel();
+      });
+
+      describe('when the pto is not defined', () => {
+        beforeEach('get the rto', () => {
```

Comment:
Since only one test use this, i think it's fine to have the small setup within the test, instead of creating a describe block for it

Reply:
Will refactor this.